integration-tests/oidc-client-wiremock/src/test/java/io/quarkus/it/keycloak/KeycloakRealmResourceManager.java

```diff
@@ -67,6 +67,45 @@ public Map<String, String> start() {
                         .withBody(
                                 "{\"access_token\":\"temp_access_token\", \"expires_in\":4}")));
 
+        server.stubFor(WireMock.post("/ciba-token")
+                .withRequestBody(matching(
+                        "grant_type=urn:openid:params:grant-type:ciba&client_id=quarkus-app&client_secret=secret&auth_req_id=16cdaa49-9591-4b63-b188-703fa3b25031"))
```

Comment:
@sberyozkin For some reason, `client_id` and `client_secret` are not sent from `oidc-client`. If I remove them from here, this test passes.

Reply:
I see now. It sends it as BASIC auth.